File: plus_minus/plus_minus.js
function main() {
  var n = parseInt(readLine());
  arr = readLine().split(' ');
  arr = arr.map(Number);
  var negative = 0;
  var zero = 0;
  var positive = 0;
  arr.forEach(num => {
    if (num < 0) {
      negative += 1;
    } else if (num === 0) {
      zero += 1;
    } else {
      positive += 1;
    }
  });
  console.log(`${(positive/n).toFixed(3)}\n${(negative/n).toFixed(3)}\n${(zero/n).toFixed(3)}`)
}

File: plus_minus/plus_minus.py
import sys

n = int(input().strip())
arr = [int(arr_temp) for arr_temp in input().strip().split(' ')]
negative =0
zero = 0
positive = 0
for x in arr:
    if x < 0:
        negative += 1
    elif x == 0:
        zero += 1
    else:
        positive += 1

print (round((positive/n), 3), round((negative/n), 3), round((zero/n), 3), sep='\n')
